packages/react/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,38 @@
+import { CSSProperties, MouseEvent, ReactNode } from 'react';
+import { X } from "phosphor-react";
+import '@compassion-gds/css/src/components/Modal/modal.css';
+
+export interface ModalProps {
+  title?: string,
+  size?: 'default' | 'small' | 'wide',
+  footerSlot?: ReactNode,
+  children: ReactNode,
+  onClose: () => void,
+  className?: string,
+  style?: CSSProperties,
+}
+
+const Modal = ({ title = '', size = 'default', onClose, footerSlot, className, style, children }: ModalProps) => {
+    const classNames = ['modal', size, className].join(' ');
+
+    return (
+      <div className={classNames} style={style}>
+        {onClose && <ModalClose onClick={onClose} />}
+        {title.length > 0 && <div className="modal-header">{title}</div>}
```

Comment:
Why do you always use the `.length > 0` check? react-native 😅?

```jsx
{title && <div className="modal-header">{title}</div>}
```

should work as well

Reply:
JS :D